manim/mobject/svg/text_mobject.py

```diff
@@ -575,3 +578,331 @@ def change_alignment_for_a_line(self, alignment, line_no):
+
+    Examples
+    --------
+    .. manim:: PangoRender
+        :quality: low
+
+        class PangoRender(Scene):
+            def construct(self):
+                morning = PangoText("வணக்கம்", font="sans-serif")
+                self.play(Write(morning))
+                self.wait(2)
+
+    .. WARNING::
+
+        Using a :class:`.Transform` on text with leading whitespace can look
+        `weird <https://github.com/3b1b/manim/issues/1067>`_. Consider using
+        :meth:`remove_invisible_chars` to resolve this issue.
+
+    """
+
+    CONFIG = {
+        # Mobject
+        "color": WHITE,
+        "height": None,
+        "width": None,
+        "fill_opacity": 1,
+        "stroke_width": 0,
+        "should_center": True,
+        "unpack_groups": True,
+        # Text
+        "font": "",
+        "gradient": None,
+        "line_spacing": -1,
+        "size": 1,
+        "slant": NORMAL,
+        "weight": NORMAL,
+        "t2c": {},
+        "t2f": {},
+        "t2g": {},
+        "t2s": {},
+        "t2w": {},
+        "tab_width": 4,
+    }
+
+    def __init__(self, text: str, **config):
+        self.full2short(config)
+        digest_config(self, config)
+        self.original_text = text
+        text_without_tabs = text
+        if text.find("\t") != -1:
+            text_without_tabs = text.replace("\t", " " * self.tab_width)
+        self.text = text_without_tabs
+        if self.line_spacing == -1:
+            self.line_spacing = self.size + self.size * 0.3
+        else:
+            self.line_spacing = self.size + self.size * self.line_spacing
+        file_name = self.text2svg()
+        self.remove_last_M(file_name)
+        SVGMobject.__init__(self, file_name, **config)
+        self.text = text
+        self.chars = VGroup(*self.submobjects)
+        self.text = text_without_tabs.replace(" ", "").replace("\n", "")
+        nppc = self.n_points_per_cubic_curve
+        for each in self:
+            if len(each.points) == 0:
+                continue
+            points = each.points
+            last = points[0]
+            each.clear_points()
+            for index, point in enumerate(points):
+                each.append_points([point])
+                if (
+                    index != len(points) - 1
+                    and (index + 1) % nppc == 0
+                    and any(point != points[index + 1])
+                ):
+                    each.add_line_to(last)
+                    last = points[index + 1]
+            each.add_line_to(last)
+        if self.t2c:
+            self.set_color_by_t2c()
+        if self.gradient:
+            self.set_color_by_gradient(*self.gradient)
+        if self.t2g:
+            self.set_color_by_t2g()
+        # anti-aliasing
+        if self.height is None and self.width is None:
+            self.scale(TEXT_MOB_SCALE_FACTOR)
+
+    def remove_last_M(self, file_name: str):
+        """Internally used function. Use to format the rendered SVG files."""
```

Comment:
Nitpick: there's no need to mention in the docstring that this is a function.

```suggestion
        """Internally used. Use to format the rendered SVG files."""
```